### ISSUE TITLE
Create Landingpage of Apache-Pulsar-UI

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#36]
## User story
1. As a user
2. I want a basic Landing-page which provides a possibility to set the Apache Pulsar Endpoint
3. So that I can specify to which Apache Pulsar Instance I want to connect to

## Acceptance criteria
* [ ] Input that takes the hostname of the Apache Pulsar Endpoint (and stores it in the frontendlogic to pass it to the backend later on)
=> Use the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) inspiration
* [ ] When the "connect button" is pressed the Page should redirect to the Cluster View

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
